components/data-table/index.jsx

```diff
@@ -316,14 +316,10 @@ class DataTable extends React.Component {
 					}
 
 					const cellFixed = column.querySelector('.slds-cell-fixed');
-					const linkFixed = cellFixed.firstChild;
 
 					if (cellFixed) {
 						cellFixed.style.left = `${columnLeft - wrapperLeft}px`;
-					}
-
-					if (linkFixed) {
```

Comment:
@interactivellama why was this altered? I had styles to make headers fill the cellFixed element provided it was properly sized. If there was something I missed I would prefer to add `width: 100%` to whatever it was and only resize the cellFixed element rather than attempt to size it's immediate child manually in this way

Reply:
I believe the value was null in the radio group variant which resulted in a key call on an undefined object.

https://github.com/salesforce/design-system-react/issues/1708

Reply:
Resolved via meeting